src/main/java/org/influxdb/impl/InfluxDBResultMapper.java

```diff
@@ -152,7 +202,7 @@ void throwExceptionIfResultWithError(final QueryResult queryResult) {
     });
   }
 
-  void cacheMeasurementClass(final Class<?>... classVarAgrs) {
+  public void cacheMeasurementClass(final Class<?>... classVarAgrs) {
```

Comment:
Please revert the access modifier from this method on this class and from all methods from other classes to the minimum required. There is no need to make this method public.